src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/settings/author-selection/main.js

```diff
@@ -68,51 +83,67 @@ define(['jquery', 'text!/admin/api/contacts/fields'], function($, fieldsResponse
 
         initialize: function() {
             this.data = this.options.data;
-
             this.bindCustomEvents();
+
+            var authored = this.options.data.authored ? this.options.data.authored : null;
+            var authoredTime = authored
+                ? Globalize.format(new Date(authored), Globalize.culture().calendar.patterns.t)
+                : null;
+
             this.html($(this.templates.skeleton({
                 translations: this.translations,
                 nullableAuthor: this.options.nullableAuthor,
-                authored: this.options.data.authored ? this.options.data.authored : null
```

Comment:
the authored field will be empty with this change.

Reply:
This is not required anymore since the field value will be set by form mapper (`this.sandbox.form.setData()`).
However, form mapper fails to set initial value for authoredTime field so I've left that one intact.